markdown/1.x/en/ff-record-list.md

```diff
@@ -128,6 +128,9 @@ Note: When you manually load a page, e.g. from an URL with the 3rd page set in i
 
 We recommend using higher values for `[infinite-debounce-delay]` and `infinite-scroll-margin` like:
 
+**Note:**
```

Comment:
The _Note_ is probably best placed below the code listing. The `...like:` from above doesn't work anymore.